symex-evil.el

```diff
@@ -123,7 +123,7 @@ executing this command to get the expected behavior."
     ("d" . symex-evaluate-definition)
     ("M-e" . symex-eval-recursive)
     ("T" . symex-evaluate-thunk)
-    (":" . eval-expression)
+    (":" . evil-ex)
```

Comment:
What happens if you leave this binding out entirely? I believe it would just pass through to the Normal mode binding, i.e. what we want.

Reply:
that works too!